core/src/main/java/com/graphhopper/routing/weighting/TurnWeighting.java

```diff
@@ -56,16 +56,16 @@ public TurnWeighting(Weighting superWeighting, TurnCostExtension turnCostExt) {
      * Set the default cost for an u-turn in seconds. Default is 40s. Should be that high to avoid
      * 'tricking' other turn costs or restrictions.
      */
-    public TurnWeighting setDefaultUTurnCost(double costInSeconds) {
-        defaultUTurnCost = costInSeconds;
+    public TurnWeighting setUTurnCost(double costInSeconds) {
+        uTurnCost = costInSeconds;
```

Comment:
Maybe we remove this setter and move it to a new constructor?

Reply:
ok yes good idea, done.